docs/user-guide/creating-application/app-details.md

```diff
@@ -41,4 +41,25 @@ Click **Save**. The application will be moved to the selected project.
 
 **Note**: If you change the project:
 * The current users will lose the access to the application.
-* The users who already have an access to the selected project, will get an access to the application automatically.
\ No newline at end of file
+* The users who already have an access to the selected project, will get an access to the application automatically.
+
+
+## Ingress Host URL
+
+You can copy the Ingress Host URL from the **URLs** on the **App Details** instead of searching in the `Manifest`.
```

Comment:
Can be changed to: 
Ingress Host URL or Load Balancer URLs can now be seen in **URLs** section on the **App Details**